README.md

```diff
@@ -70,11 +70,11 @@ If developing the SDK, you also need:
 
 ## Getting Started
 
-Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started.html) for an example of authenticating and making an API call using the SDK.
+Visit the [Getting Started Guide](https://carbon-black-cloud-python-sdk.readthedocs.io/en/latest/getting-started) for an example of authenticating and making an API call using the SDK.
 
```

Comment:
After the bullet points for products at lines 37-44, and a note that support for newer products and features such as Workload are on the roadmap/coming soon.

Reply:
Added a note let me know what you think